ksqldb-engine/src/main/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporter.java

```diff
@@ -49,38 +48,30 @@
       = LoggerFactory.getLogger(StorageUtilizationMetricsReporter.class);
   private static final String METRIC_GROUP = "ksqldb_utilization";
 
-  private final Map<String, Map<String, TaskStorageMetric>> metricsSeen;
-  private final Metrics metricRegistry;
-  static AtomicBoolean registeredNodeMetrics = new AtomicBoolean(false);
```

Comment:
It looks like this flag was a hack to fix some of the tests that were clashing on the static metrics registry. It's not needed anymore.